### PR TITLE
Cosmetic fixes for linux packaging

### PR DESCRIPTION
**Pull request by allonsy**
Some quick cosmetic fixes in the readme files

### CODE REVIEW

File: package/packaging.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
-# Git-Town Packaging and Release Process
+# Git-Town Packaging and Release Process For Linux
```
Comment (charlierudolph) [inline package/packaging.md]: Linux is already a subheader on line 5. Can that subheader be removed?

Comment (allonsy) [inline package/packaging.md]: Good catch, new version is pushed

File: package/packaging.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
-Instructions on how to update all the necessary packages once a new release is out
+This document contains instructions on how to update all the necessary packages once a new release is out.
 
-## Linux
 The current linux (all distros) maintainer is [allonsy](https://github.com/allonsy). If an update occurs, please bug him
+
 ### Arch Linux
 The package will need to be flagged out of date [here](https://aur.archlinux.org/packages/git-town/)
 This will tell [allonsy](https://github.com/allonsy) to update the package (most likely minimal intervention necessary). There is a button on the right hand side menu to flag the package.
```
Comment (charlierudolph) [inline package/packaging.md]: While you're updating this, can you please add periods to these sentences? They end up grouped together. Same for sentences under `Debian Based Distributions`

Comment (charlierudolph) [inline package/packaging.md]: Actually looks like line 8 is the only one that needs a period.

**Comment by allonsy** [comment]
Added punctuation

**Comment by charlierudolph** [comment]
Looks good to me. :ship: (using git-ship or using github's new [squash merge UI](https://github.com/blog/2141-squash-your-commits))

**Pull request closed without merging**